test/cql-pytest/test_compaction_strategy_validation.py

```diff
@@ -50,8 +50,9 @@ def test_leveled_compaction_strategy_options(cql, table1):
 def test_not_allowed_options(cql, table1):
     def scylla_error(**kwargs):
         template = "Invalid compaction strategy options {{{}}} for chosen strategy type"
-        options = ', '.join(f"{{{k}, {v}}}" for k, v in kwargs.items())
-        return template.format(options)
+        options1 = ', '.join(f"{{{k}, {v}}}" for k, v in kwargs.items())
```

Comment:
nitpick: can you please add a comment explaining why these two versions exist, even saying that options1 is the old one and can eventually be removed? You can even call the new one "options" and the old one "old_options" to make this even more glaring.
(yes, as you said in the commit message we can revert this change, but it is likely we'll never remember to do it, so we can have a comment in the code to remind us to do it.

Reply:
sure.